Readme.md

```diff
@@ -56,13 +56,23 @@ result.map // source map object
 
 ### Errors
 
-Errors will have `error.position`, just like [`node.position`](#position). The
-error contains the source position in the message. To get the error message
-without the position use `error.reason`.
+Errors thrown during parsing have the following properties:
+
+- `message`: `String`. The full error message with the source position.
+- `reason`: `String`. The error message without position.
+- `filename`: `String` or `undefined`. The value of `options.source` if
+  passed to `css.parse`. Otherwise `undefined`.
+- `line`: `Integer`.
+- `column`: `Integer`.
+- `source`: `String`. The portion of code that couldn't be parsed.
```

Comment:
Looks good! Just remove `` ` ``s around the property names, because of commit 40098815.